### DISCUSSION TITLE
Select Custom Type

### DISCUSSION THREAD

**Discussion by niitou** [SeaQL/sea-orm#1617] [Q&A] [answered]
I have a custom composite type for my project that look like this 
![image](https://user-images.githubusercontent.com/57691998/234745767-85d0a33a-ab6d-41d6-9b6d-f6c627542483.png)

Using sea-orm-cli, the generated entity is looking like this
![image](https://user-images.githubusercontent.com/57691998/234747399-3c5cb918-ab1d-49d1-803f-93957fde3548.png)

My problem is I try to select data from my database it always generate this error
`
DbErr: Query Error: error occurred while decoding column "title": mismatched types; Rust type 'core::option::Option<alloc::string::String>' (as SQL type 'TEXT') is not compatible with SQL type 'media_title'
`

I tried to change the data type on my entity to Json but it still gives me the same error. And I tried to pass custom type on the entity and it gives me a different error.
![image](https://user-images.githubusercontent.com/57691998/234747217-8e0e8cdc-7a9f-4dc1-8c1f-11ff16dac383.png)

Is there a way to select my custom type ?

**Comment by tyt2y3** [comment, marked as answer]
I don't think there is work done to support composite types across the SeaQL libraries.

**Reply by niitou** [reply]
Oh okay, thanks for the answer. Guess I have to change that  column type to Json instead of composite.
